README.md

```diff
@@ -35,6 +35,17 @@ http://ecotrust-canada.github.io/markdown-toc/ -->
   v10.x+ accessible from the target node
 - Ansible >= 2.9
 
+## Using ansible-conjur-collection with Conjur OSS
+
+Are you using this project with [Conjur OSS](https://github.com/cyberark/conjur)? Then we
+**strongly** recommend choosing the version of this project to use from the latest [Conjur OSS
+suite release](https://docs.conjur.org/Latest/en/Content/Overview/Conjur-OSS-Suite-Overview.html).
```

Comment:
Would it be better to link directly to the Ansible-specific section of the Suite release docs? i.e.
https://docs.conjur.org/Latest/en/Content/Integrations/ansible.html

Reply:
in this case we are linking to the suite release stuff, not the ansible integration page. the version to use shows only in the suite release notes.

Reply:
Maybe the same link used below for `latest suite release` (https://docs.conjur.org/Latest/en/Content/ReleaseNotes/ConjurOSS-suite-RN.htm) would make it clearer what the latest suite release is? From the Release notes page, they can always navigate up to the overview.